src/main/java/cubicchunks/asm/mixin/core/common/MixinWorld.java

```diff
@@ -215,6 +215,13 @@ public boolean testForCubes(CubePos start, CubePos end, Predicate<Cube> cubeAllo
     @Override public Profiler getProfiler() {
         return this.theProfiler;
     }
+    
+    //Overwriting original World.markChunkDirty()
+    public void markChunkDirty(BlockPos pos, TileEntity unusedTileEntity) {
```

Comment:
So add it. Without that annotation it will maybe work in dev environment but will break on obfuscated environment - method with that name won't exist.